CONTRIBUTING.md

```diff
@@ -237,24 +237,35 @@ request. Typically reviewers will review the code for obvious errors, as well as
 test out the patch set and opine on the technical merits of the patch. Project
 maintainers take into account the peer review when determining if there is
 consensus to merge a pull request (remember that discussions may have been
-spread out over GitHub, mailing list and IRC discussions). The following
+spread out over GitHub, mailing list and IRC discussions).
+
+#### Conceptual Review
+
+A review can be a conceptual review, where the reviewer leaves a comment
+ * `Concept (N)ACK`, meaning "I do (not) agree in the general goal of this pull
+   request",
+ * `Approach (N)ACK`, meaning `Concept ACK` and "I also do (not) agree with the
+   approach of this change".
+
+A `NACK` needs to include a rationale why the change is or is not worthwhile.
```

Comment:
I think you could drop the "is or" from that sentence

Reply:
Done